torch/_inductor/codegen/common.py

```diff
@@ -98,6 +98,8 @@ def deduce_node_dtype_by_subgraph(self, node: torch.fx.Node):
         return dtype
 
     def deduce_node_dtype(self, node: torch.fx.Node):
+        from .cpp import DTYPE_TO_COMPUTATION_DTYPE
```

Comment:
It is weird `common.py` depends on something from `cpp.py`. If it is a common module, shall we move `DTYPE_TO_COMPUTATION_DTYPE` here too?

Reply:
Done, moved it to `common.py`.